devel/GitUp/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            git-up GitUp 1.2 v
+
+checksums               rmd160  c8c5eab4cf5bfed8abe934d57ea34cb8b8b769f4 \
+                        sha256  3b8934cfd72f6fad899bf95e7e5b983e6a3543e2ce2226005463b5e7f872e216 \
+                        size    21309896
+
+categories              devel
+platforms               darwin
+license                 GPL-3
+maintainers             {@knapoc knapoc} openmaintainer
+description             GitUp is a native macOS git client
+long_description        GitUp provides features such as a live and interactive repo graph \
+                        unlimited redo/undo, snapshots for 1-click rollbacks, visual commit \
+                        splitter and many more
+homepage                https://gitup.co
+
+
+fetch.type              git
+
+post-fetch {
+    system -W ${worksrcpath} "git submodule update --init"
+}
+
+patchfiles              patch-disable-sparkle.diff \
+                        patch-disable-clt-install.diff
+
+use_configure           no
+use_xcode               yes
+
+variant clt description {Installs GitUp command line tool} {
+}
+
+default_variants        +clt
+
+build {
+    system -W ${build.dir}/GitUp "xcodebuild -derivedDataPath ${build.dir} -configuration Release -scheme Application 'CODE_SIGN_IDENTITY=' CODE_SIGNING_REQUIRED=NO 'CODE_SIGN_ENTITLEMENTS='"
+}
```

Comment:
There is an [Xcode PortGroup](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/xcode-1.0.tcl). I think it's preferred to use it here.

Reply:
I tried to run into various issues while using the Xcode PortGroup. I believe this setup allows for easier maintainability.